src/Provider/Vkontakte.php

```diff
@@ -134,6 +137,7 @@ public function getUserProfile()
 
         $userProfile->data = [
             'education' => $data->get('education'),
+            'bdate' => $data->get('bdate')
```

Comment:
we have such fields like birthDay / birthMonth / birthYear. Let's use them (see https://github.com/hybridauth/hybridauth/blob/master/src/User/Profile.php#L99)

Reply:
This fields are empty.

![troubleVK3](https://user-images.githubusercontent.com/1305838/77416773-24787080-6de6-11ea-8424-d1a6a0cecfb2.jpg)

It was a temporary solution.

Reply:
it's empty, because there are no code to fill them. Pls extend it to use these fields instead of $userProfile->data['bday']

Reply:
```php
$adapter->getUserProfile()->birthDay;
```
Right?

Reply:
once more - you need to fill these fields like

```php
$bday = explode('.', $data->get('bdate'));
$userProfile->birthDay = (int) $result[0];
$userProfile->birthMonth = (int) $result[1];
$userProfile->birthYear = (int) $result[2];
```


and finally, `$adapter->getUserProfile()->birthDay;` will work

Reply:
Wow. Thank you :) I didn`t know about it.

Reply:
I`m confused. Where should I do it?

Reply:
I`m tried do this on $adapter, but I got an error:

Fatal error: Uncaught Error: Call to undefined method Hybridauth\Provider\Vkontakte::get()